integrations/criteo/lib/index.js

```diff
@@ -253,3 +254,23 @@ function getProductMetadata(track) {
 
   return products;
 }
+
+/**
+ * Get Device Type
+ *
+ * @api private
+ *
+ * REF: https://support.criteo.com/s/article?article=202806931-Managing-your-different-site-types&language=en_US
+ */
+
+function getDeviceType() {
+  var ipadRegex = /iPad/;
+  var mobileRegex = /Mobile|iP(hone|od)|Android|BlackBerry|IEMobile|Silk/;
```

Comment:
should it not be `iP(hone|ad)` for iPhone or iPad?Are we catering to users agents on the iPod?

Reply:
nvm we have the iPad regex separately.but iPod?

Reply:
This regex referred from criteo's docs only.. see the reference link

Reply:
gotcha.